suites/pacific/rbd/tier-2_rbd_regression.yaml

```diff
@@ -121,3 +121,45 @@ tests:
       module: trash_restore.py
       name: Check trash restore functionality
       polarion-id: CEPH-83573298
+
+  - test:
+      config:
+        ec-pool-k-m: 2,1
```

Comment:
can we keep ec-pool k and m other than default value 2,1 as discussed?

Reply:
I would prefer to have it as default value itself for now, since I am not sure of the functionality of k and m values. If you feel there is a specific scenario where non default values need to be used we can add the same as a polarion test case and take up automation in a different PR for the same.

Also I think @VasishtaShastry is working on setting some dummy value other than 2,1 to facilitate using ec pool without having to create erasure profile. Vasi we could add that as a different PR once your PR for the default value changes is done right?

Reply:
@sunilangadi2 I'm not able to recall conversation in which we decided to try different values of k,m

Can you please elaborate why RBD FG need to focus on trying out different combination of erasure code ?
Also, I don't see any blocking concern for this PR, we could take the need and conversation offline with a new tracker as this PR is quite old.
Please let me know your thoughts.

Reply:
ok sure @manasagowri @VasishtaShastry marking it as resolved.